deploy/kubernetes/resources/gameon-impl-service-akka-remoting.json

```diff
@@ -0,0 +1,22 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "gameonservice-99"
```

Comment:
@mckeeh3 I'd like to modify this slightly to use "gameon17s99" consistently as the service name everywhere. It will make it much more straightforward to be consistent with naming.